### ISSUE TITLE
Application Crash on OSX 10.11.6

### ISSUE THREAD

**Issue by sfeary** [keepassxreboot/keepassxc#265]
Crashes immediately after launch every time. Excerpt from the crash report points to a dynamic library compiled for 10.12:

```
Application Specific Information:
/Volumes/KeePassXC/KeePassXC.app/Contents/MacOS/../Frameworks/libmicrohttpd.12.dylib

Dyld Error Message:
  Symbol not found: _clock_gettime
  Referenced from: /Volumes/VOLUME/KeePassXC.app/Contents/MacOS/../Frameworks/libmicrohttpd.12.dylib (which was built for Mac OS X 10.12)
  Expected in: /usr/lib/libSystem.B.dylib
```

## Your Environment
* KeePassXC version/commit used: 2.1.1
* Operating System and version: OSX 10.11.6

**Comment by TheZ3ro** [comment]
Thanks, we are already working on this, on the website the latest version available is 2.1.0, use that instead.

**Comment by phoerious** [comment]
I rebuilt the DMG, can you please test https://github.com/keepassxreboot/keepassxc/releases/download/2.1.1/KeePassXC-2.1.1.dmg ?
Signature: https://github.com/keepassxreboot/keepassxc/releases/download/2.1.1/KeePassXC-2.1.1.dmg.sig

**Comment by sfeary** [comment]
No unfortunately it's the same error with the new version as well.

**Comment by phoerious** [comment]
Weird. So apparently MACOSX_DEPLOYMENT_TARGET=10.7 wasn't properly populated to my Macports libmicrohttpd build.

**Comment by phoerious** [comment]
Alright, I rebuilt libmicrohttpd by hand and made sure the CFLAGS were properly set. Please try again. If it fails again, I guess I have to do the same for KeePassXC.

**Comment by sfeary** [comment]
Sorry to report it failed again.

**Comment by phoerious** [comment]
Okay, last try @sfeary.
If that doesn't work now, we have to move this to 2.1.2. droidmonkey is working on getting rid of libmicrohttpd.

**Comment by sfeary** [comment]
Sorry still crashes out on launch.

**Comment by phoerious** [comment]
Okay, I guess this problem cannot be solved with a simple compiler flag. I'm afraid we have to move this to 2.1.2.

**Comment by phoerious** [comment]
I compiled two versions: one with KeePassHTTP and one without. The one without should also run on OS X < 10.12. Please try: https://keepassxc.org/download#mac

**Comment by sfeary** [comment]
It works, thanks!

**Comment by ehiggs** [comment]
Any idea when this will land in homebrew?

**Comment by phoerious** [comment]
We are in no direct contact to the homebrew packager, but I would assume next version. We got rid of libmicrohttpd, so we will be able to ship only one package which should run on all Macs.

**Comment by TheZ3ro** [comment]
@ehiggs we currently are on [homebrew cask](https://caskroom.github.io/) that automatically download the latest release available for KeePassXC with HTTP enabled.

**Comment by phoerious** [comment]
Right , that's what I meant. I have no idea when and if we will be on homebrew directly. But the cask is currently only with HTTP. The next version should solve that.

**Comment by phoerious** [comment]
Will be fixed in 2.1.2.
